frc46_factory_token/token_impl/src/lib.rs

```diff
@@ -99,24 +108,19 @@ pub struct ConstructorParams {
 }
 
 pub fn construct_token(params: ConstructorParams) -> Result<u32, RuntimeError> {
-    let runtime = ActorRuntime::<FvmSyscalls, Blockstore>::new_fvm_runtime();
+    let runtime = ActorRuntime::<FvmSyscalls, RuntimeBlockstore>::new_fvm_runtime();
```

Comment:
It seems wrong that the FvmSyscalls is used concretely here, rather than just the trait. Shouldn't the runtime be passed in as a parameter?

Reply:
I'll pass in the `ActorRuntime` since it does look weird here. It's still only used in actor code though because it returns a block ID. It had to live here in the past to avoid import conflicts in the integration tests, but that's also fixed now.